### PR TITLE
feat(items): add support for arrays

### PR DESCRIPTION
**Pull request by kentcdodds**
With this commit you can now specify an array of items to rank and those items will be checked individually (before, they were joined with a comma via toString, which is sub optimal).

@conorhastings, I need this for what I'm working on right now, so if you could review this really quick that'd be awesome. If you don't have time no worries. I'll probably not need it for an hour. If you don't get to it I'll just merge it myself :)

### CODE REVIEW

**Comment by kentcdodds** [comment]
LGTM

**Comment by kentcdodds** [comment]
Also, it looks like lgtm.co is working now! 🎉

**Comment by conorhastings** [comment]
@kentcdodds looks good to me! This is a pretty cool feature!

File: src/index.test.js [hunk 1]
```diff
@@ -130,6 +130,35 @@ const tests = {
...
+      ],
+      'cc',
+      {keys: ['favoriteIceCream']},
+    ],
+    output: [
+      {favoriteIceCream: ['candy cane', 'brownie']},
+      {favoriteIceCream: ['mint', 'chocolate']},
+    ],
+  },
+  'when using arrays of values, when things are equal, the one with the higher index wins': {
```
Comment (kentcdodds) [inline src/index.test.js]: This kinda just fell out of the implementation. Pretty solid actually 👍

**Comment by kentcdodds** [comment]
Thanks @conorhastings!

**Pull request merged**